include/orioledb.h

```diff
@@ -128,10 +128,27 @@ typedef enum
 	oIndexToast = 1,
 	oIndexBridge = 2,
 	oIndexPrimary = 3,
-	oIndexUnique = 4,
-	oIndexRegular = 5,
+	oIndexExclusion = 4,
```

Comment:
Is there a specific reason why oIndexExclusion should come before oIndexUnique?  I think this complicates the binary update of SYS_TREES_O_INDICES.  If we even need to have this specific order for checkpointing, I think it would be better to add a new value to the end of the enum and adjust the comparison function.